index.Rmd

```diff
@@ -295,18 +295,19 @@ plot1=ggplot(data = All_groupings,aes(x=Year,y=percpos,color=Pathogen)) +
         ylab("Pathogen Prevalence (%)")+
         scale_x_continuous(breaks=c(2008:2019))
 ```
-
+ 
 The code to create an interactive map where each clicking each county displays a time-series graph comparing county-level pathogen rates to the NYS average is shown below:
 ```{r interactive, echo=TRUE,warning=FALSE,message=FALSE,error=F}
 County_Boundaries=st_transform(County_Boundaries,CRS("+proj=longlat +datum=WGS84"))
 names(County_Boundaries)[1]="County"
 test_new=left_join(County_Boundaries,test2,by='County')
 plot2=leaflet() %>%
-        addTiles() %>%
+        addTiles() %>% 
         addPolygons(data=test_new,
                     group='County',
                     weight=1,
                     label=~County,
+                    color= "different_color_you_want",
```

Comment:
I noticed your polygons color is blue, which reminds me of water and almost blends in with Lake Erie and Lake Ontario. This could be especially troubling to people that are color blind. Try different color options to avoid using blue to see what looks best to you.

Reply:
Good call, I'll go back and fix this!